src/tests/ftest/util/soak_utils.py

```diff
@@ -1219,7 +1223,7 @@ def create_app_cmdline(self, job_spec, pool, ppn, nodesperjob):
                 sbatch_cmds.extend(dfuse_start_cmdlist)
             # allow apps that use an mpi other than default (self.mpi_module)
             if mpi_module != self.mpi_module:
-                sbatch_cmds.append(f"module load {mpi_module}")
+                sbatch_cmds.append(f"module use {mpi_module_use};module load {mpi_module}")
```

Comment:
Could this actually just be moved above? to line 1216?

Reply:
There is a code change needed here and I will address that too.  The self.mpi_module_use and self.mpi_module are only necessary if using dfuse.  So I will fix that.

Also, I originally had tried what you are suggesting i.e use the same mpi lib to launch dfuse and run the app but I could not get dfuse to run with IntelMPI.   So I launched it with self.mpi_module*  I need to re-run using mpi_module* scenario and verify if it can run.